<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>640</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>2</red>
        <green>195</green>
        <blue>154</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>249</green>
        <blue>222</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>121</green>
        <blue>94</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>162</green>
        <blue>126</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>5</red>
        <green>102</green>
        <blue>141</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>243</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>249</green>
        <blue>222</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="127">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>227</red>
        <green>227</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>105</red>
        <green>105</green>
        <blue>105</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>245</red>
        <green>245</green>
        <blue>245</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>121</green>
        <blue>94</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>2</red>
        <green>195</green>
        <blue>154</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>249</green>
        <blue>222</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>121</green>
        <blue>94</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>162</green>
        <blue>126</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>121</green>
        <blue>94</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>121</green>
        <blue>94</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>243</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>243</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>245</red>
        <green>245</green>
        <blue>245</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Добро пожаловать в генератор случайных чисел. Пожалуйста выберите метод</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>331</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>1) Линейно-конгруэтный метод</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>381</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Пожалуйста, выберите диапозон чисел:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>От 0 до:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="values_LK">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>93</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>80</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Выберите количество чисел:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>n =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="count_LK">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>110</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="statistics_LK">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>291</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Высчитать статистику метода</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>311</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>2) Треугольное распределение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Пожалуйста, выберите диапозон чисел:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>От</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_values_Tr">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>до</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_values_Tr">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Пиковое значение:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="peak_value_Tr">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>250</y>
      <width>281</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Выберите количество чисел:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>280</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>n =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="count_Tr">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>280</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="statistics_Tr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>291</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Высчитать статистику метода</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>420</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>3) Распределение по времени</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>381</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Пожалуйста, выберите диапозон чисел:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>От 0 до:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="values_Time">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>93</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>460</y>
      <width>281</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Выберите количество чисел:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>490</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>n =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="count_Time">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>490</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="statistics_Time">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>291</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <property name="text">
     <string>Высчитать статистику метода</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
